test/regressions/github-1136-test

```diff
@@ -13,21 +13,54 @@ test_case bang-cmdlineargs-doesnt-crash \
 	--script='
 	:!%(cmdlineargs)
 	' <<EOF
+On branch master
+Changes to be committed:
+  (no files)
+Changes not staged for commit:
+  (no files)
+Untracked files:
+  (no files)
 
+[status] Nothing to update                                                  100%
```

Comment:
This is not consistent with the comment "This runs an empty command, hence the empty pager".